app/models/post.rb

```diff
@@ -0,0 +1,23 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'post_id'
+  has_many :comments, foreign_key: 'post_id'
```

Comment:
- [ OPTIONAL ] Great job so far @vanina123 There are just a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink:
- [ ] I will suggest that you consider making use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html). :+1: In this case, you can make use of  the `after_save` callback to call the `#post_counter` method every time a new post has been saved to the database, then make the `increase_post_counter` method private :+1:  
- The `after_save` callback, is triggered every time an object is saved to the database, regardless of whether it is a new record or an existing one being updated. It is invoked both after the creation of a new record and after updating an existing record and because of the nature of the blog app and the purpose of this `post_counter` method, it will be efficient to use it in your  Project :+1: 
- I will also suggest you use that in the other models as well :wink: 

Example Code
```ruby
  # app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end

```